Add the `COE 1v1 Tournament: Registrations Are Live!` news post and `COE 2024` wiki page (#11744)
ppy/osu-wiki: news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md

---
layout: post
title: "COE X YUKI AIM 2024 1v1 Tournament: Registrations are live!"
date: 2024-07-17 20:00:00 +0000
---

It has never been a better time to showcase your osu skills on LAN than the annual COE 1v1 tournament, now with Yuki Aim as a partner!

![](/wiki/shared/news/2024-07-17-coe-x-yukiaim-1v1-tournament/banner.jpg)

> If you are somehow not familiar COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe%27s_osu%21_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) and [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!

Author: ```suggestion
If you are somehow not familiar with COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe's_osu!_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) and [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!
```

> If you are somehow not familiar with COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe's_osu!_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) and [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!

Reviewer: ```suggestion
If you are somehow not familiar with COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe's_osu!_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) & [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!
```

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/X6S5LuYHywk" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

The registrations for the tournament are live now! As per usual, the COE 1v1 LAN tournament will have 16 best players from the online qualifiers duel it out over the course of 3 days in a single elimination bracket, with all matches being played on the new [osu!(lazer)](/wiki/Client/Release_stream/Lazer) client.

### Tournament schedule

| Event | Timestamp (UTC) |
| --: | :-- |
| Round of 16 | 2024-08-01 (09:00–21:00) |
| Quarterfinals | 2024-08-02 (11:30–18:00) |
| Semifinals | 2024-08-03 (10:30–13:00) |
| Showmatches | 2024-08-03 (13:30–14:30) |
| Finals | 2024-08-03 (15:30–16:30) |
| Grand Finals | 2024-08-03 (18:00–19:30) |

### Prizes

> The total prize pool of &euro;3000 will be divided among the top 3 players, along with physical prizes also being handed out.

Author: ```suggestion
The total prize pool of €3000 will be divided among the top 3 players, along with physical prizes also being handed out.
```
